config/config_string.go

```diff
@@ -148,6 +148,42 @@ func outputsStr(os []*Output) string {
 	return strings.TrimSpace(result)
 }
 
+func localsStr(ls []*Local) string {
```

Comment:
I'm honestly not sure why these are here vs. in a test-only file, but I was following the precedent set by the others above.